src/CrowdedMod/Patches/CreateGameOptionsPatches.cs

```diff
@@ -30,7 +30,7 @@ public static void Postfix(CreateOptionsPicker __instance)
                             var playerButton = __instance.MaxPlayerButtons[i];
 
                             var tmp = playerButton.GetComponentInChildren<TextMeshPro>();
-                            var newValue = Mathf.Max(byte.Parse(tmp.text) - 10, byte.Parse(playerButton.name));
+                            var newValue = Mathf.Max(byte.Parse(tmp.text) - 10, byte.Parse(playerButton.name) - 2);
```

Comment:
I'd stick to 4 tbh, but 3 is fine I guess

Reply:
honestly i thought of it as funny seeing a 3 player lobby, it's merely a joke-ish change which i'm willing to change to 4 if needed